source/jormungandr/tests/isochrone_tests.py

```diff
@@ -124,3 +124,12 @@ def test_isochrone_count(self):
         response = self.query(query)
         assert len(response["journeys"]) == 1
         is_valid_isochrone_response(response, self.tester, query)
+
+    def test_invalid_count(self):
+        query = "v1/coverage/basic_routing_test/stop_points/A/journeys?max_duration=25500&datetime=20120614T080000"
+        response = self.query(query)
+        assert len(response["journeys"]) == 2
+        # invalid count
+        query += "&count=toto"
+        response = self.query(query, might_have_additional_args=True)
```

Comment:
the better way to do this is self.query_no_assert I think, it returns the response + an http error code that you might check